Clarify RFC approval details in README (#38)
department-of-veterans-affairs/va.gov-platform-architecture: rfc/README.md

Whenever your team is making a decision that could affect the platform, write the details and get feedback from others on the Platform. Anyone - including product managers, UX, research, and engineers - is encouraged to create RFCs in order to solicit ideas in preparation of a decision direction.

> Note that RFC approval is _not an approval of the content_ of the RFC, but instead _reflects approval that the process was followed_. You should not feel the need to _wait for approval_ before starting your work.

Reviewer: suggest using "approval of Pull Request" to indicate that approval refers to PR process

Author: great language. I think I modified all the mentions of "approval" here, thanks for your suggestions!

> Note that RFC approval is _not an approval of the content_ of the RFC, but instead _reflects approval that the process was followed_. You should not feel the need to _wait for approval_ before starting your work.

Reviewer: ```suggestion
Note that RFC Pull Request approval is _not an approval of the content_ of the RFC, but instead _reflects approval that the process was followed_. You should not feel the need to _wait for approval_ before starting your work.
```

    * When collaborators have finished updating the RFC and it is ready for public comment, move the RFC from the "Draft" column to "Ready" on the [Platform Architecture project board](https://github.com/orgs/department-of-veterans-affairs/projects/710/views/1).
    * Update the Comment Deadline in the RFC document to an appropriate deadline. 30 days from when the RFC is moved into the "Ready" state is recommended.
1. Collect comments via the Pull Request (see also [How do I comment on an RFC](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request)) until the Comment Deadline.
1. Address and resolve every comment, all comments must be resolved before the Pull Request can be merged into `main`, to be sure everyone's contributions have been acknoledged and (possibly) incorportated. Every comment must receive a response by the author(s) in accordance with [addressing comments](#addressing-comments).

> 1. Once all comments are resolved and the comment deadline is reached, the `CODEOWNERS` will approve the PR and it will become mergeable. The Primary Author merges the RFC into the `main` branch, thereby completing the RFC.

Reviewer: ```suggestion
1. Once all comments are resolved and the comment deadline is reached, the `CODEOWNERS` will approve the Pull Request and it will become mergeable. The Primary Author merges the RFC into the `main` branch, thereby completing the RFC.
```

## Footnotes

> [^1]: Note that this approval is _not an approval of the content_ of the RFC, but instead _reflects approval that the process was followed_ as outlined in this document.

Reviewer: ```suggestion
[^1]: Note that the Pull Request approval is  _not an approval of the content_ of the RFC, but instead _reflects approval that the process was followed_ as outlined in this document.
```
